packages/core/security/core-security-server-internal/src/fips/fips.test.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { SecurityServiceConfigType } from '../utils';
+import { isFipsEnabled } from './fips';
+
+describe('fips', () => {
+  describe('#isFipsEnabled', () => {
+    let config: SecurityServiceConfigType;
+
+    beforeAll(() => {
+      config = {};
+    });
+
+    afterEach(() => {
+      config = {};
+    });
```

Comment:
NIT: I would ditch that part, given each test manually declare the config it need for its assertion.

Reply:
Fixed in https://github.com/elastic/kibana/pull/181187/commits/7b7ed010889b5072d430622be372d4af9bfee487